package.json

```diff
@@ -6,7 +6,7 @@
   "ungitPluginApiVersion": "0.2.0",
   "scripts": {
     "start": "node ./bin/ungit",
-    "test": "prettier --check . && grunt test",
+    "test": "grunt test",
```

Comment:
This change is currently in the [`prettier` commit](b8d381eab90991cea814badc4ebc7ef1030ce256). It would be better to move it into a separate commit instead, because the `prettier` commit is going to require some rebases / redos in the future.

Reply:
Oops, good catch :sweat_smile: